doc/whats_new/v1.1.rst

```diff
@@ -291,6 +291,13 @@ Changelog
   to train and pickle the model on 64 bit machine and load the model on a 32
   bit machine for prediction. :pr:`21552` by :user:`Loïc Estève <lesteve>`.
 
+:mod:`sklearn.datasets`
+...................
+
+- |Fix| The fetch_* functions are now thread safe. Data is first downloaded
+  to a temprary folder and then renamed.
```

Comment:
```suggestion
  to a temporary subfolder and then renamed.
```

Reply:
It seems sphinx is not happy with fetch_*.